pkg/node/manager/manager_test.go

```diff
@@ -246,6 +255,33 @@ func Test_AddNode_CustomNetworking(t *testing.T) {
 	assert.True(t, AreNodesEqual(mock.Manager.dataStore[nodeName], managedNode))
 }
 
+// Test adding node when custom networking is enabled but incorrect ENIConfig is defined; it should succeed
+func Test_AddNode_CustomNetworking_Incorrect_ENIConfig(t *testing.T) {
```

Comment:
Seems this can be combined into a table driven tests with the `Test_AddNode_CustomNetworking` and `Test_AddNode_CustomNetworking_NoENIConfig`, or event others. But we can refactor them later. No need to update in this PR. Maybe you can add a `TODO` comment for tracking.